pkgs/shells/xonsh/default.nix

```diff
@@ -90,5 +77,6 @@ python3Packages.buildPythonApplication rec {
 
   passthru = {
     shellPath = "/bin/xonsh";
```

Comment:
My NixOS configuration (https://github.com/SamLukeYes/nixos/commit/7bf704df6ea851ff68a3ad005f24178ee84e8e64) fails to build with this change.

```
error: /nix/store/0bq90g80r0b1s9s8ipznsh1azhsl8swp-xonsh-0.14.0 is not a shell package
```

Perhaps the wrapper package should pass through `shellPath`, I suppose.

Reply:
It does inherit `passthru` so I'm not sure why you're seeing this.

That error comes from `nixos/lib/utils.nix`. I copy pasted from there to make this minimal case:
``` nix
let
  pkgs = import ./. { };
  inherit (pkgs) lib;
  inherit (lib) types;

  toShellPath = shell:
    if types.shellPackage.check shell then
      "/run/current-system/sw${shell.shellPath}"
    else if types.package.check shell then
      throw "${shell} is not a shell package"
    else
      shell;

in toShellPath pkgs.xonsh

```
which works.

Reply:
> It does inherit `passthru` so I'm not sure why you're seeing this.

No, it doesn't.

```
nix-repl> legacyPackages.x86_64-linux.xonsh.passthru
{ }
```

> That error comes from `nixos/lib/utils.nix`. I copy pasted from there to make this minimal case:
> 
> ```nix
> let
>   pkgs = import ./. { };
>   inherit (pkgs) lib;
>   inherit (lib) types;
> 
>   toShellPath = shell:
>     if types.shellPackage.check shell then
>       "/run/current-system/sw${shell.shellPath}"
>     else if types.package.check shell then
>       throw "${shell} is not a shell package"
>     else
>       shell;
> 
> in toShellPath pkgs.xonsh
> ```
> 
> which works.

I also tried this on your branch. It throws the error.

Reply:
I forgot to push the change that I had made locally! Doh!